kuma/settings/common.py

```diff
@@ -45,6 +45,8 @@ def __call__(self, value):
 STAGING_DOMAIN = 'developer.allizom.org'
 STAGING_URL = PROTOCOL + STAGING_DOMAIN
 
+INTERACTIVE_EXAMPLES_BASE = 'https://interactive-examples.mdn.mozilla.net'
```

Comment:
This should use ``config`` so it can be set from the environment, like:

```
INTERACTIVE_EXAMPLES_BASE = config('https://interactive-examples.mdn.mozilla.net', default='https://interactive-examples.mdn.mozilla.net')
```